src/StatusAlert/index.jsx

```diff
@@ -109,7 +108,8 @@ class StatusAlert extends React.Component {
 }
 
 StatusAlert.propTypes = {
-  /** specifies the type of alert that is being diplayed. It defaults to "warning".  See the other available [bootstrap](https://v4-alpha.getbootstrap.com/components/alerts/) options. */
+  /** specifies the type of alert that is being diplayed. It defaults to "warning".
+   * See the other available [bootstrap](https://v4-alpha.getbootstrap.com/components/alerts/) options. */
```

Comment:
Not sure why the link here was to alpha version of bootstrap, please change it to https://react-bootstrap-v4.netlify.app/components/alerts/